apps-rendering/src/components/orderedList.tsx

```diff
@@ -0,0 +1,42 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
+import { remSpace } from '@guardian/src-foundations';
+import type { FC, ReactNode } from 'react';
+
+// ----- Component ----- //
+
+const styles: SerializedStyles = css`
+	list-style: none;
+	margin: ${remSpace[3]} 0;
+	padding-left: 0;
+	clear: both;
+
+	counter-reset: li;
+	> li:before {
```

Comment:
The distinction is subtle, but this is a pseudo-element so it should be `li::before` - see the "Note" here: https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements#syntax